pkg/serverless/otlp/otlp.go

```diff
@@ -0,0 +1,59 @@
+package otlp
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/DataDog/datadog-agent/pkg/config"
+	coreOtlp "github.com/DataDog/datadog-agent/pkg/otlp"
+	"github.com/DataDog/datadog-agent/pkg/serializer"
+	"github.com/DataDog/datadog-agent/pkg/util/log"
+)
+
+type ServerlessOTLPAgent struct {
```

Comment:
All of this needs to be documented.

Also, I think that this structure does not belong in `pkg/otlp`. It can live in `pkg/serverless` without issues. This code will not be reused anywhere else outside `pkg/serverless`, right?